test/git-prompt-server.test.js

```diff
@@ -82,6 +82,7 @@ describe('GitPromptServer', function() {
 
     it('preserves a provided username', async function() {
       this.timeout(10000);
+      this.retries(5);
```

Comment:
Also: I was hoping that #1068 would make these fail more informatively at least 🤔